code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -63,7 +63,7 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 	/// How many percent of the rounds are more peaceful.
 	var/peaceful_percentage = 50
 	/// If a high impact ruleset was executed. Only one will run at a time in most circumstances.
-	var/high_impact_ruleset_executed = FALSE
+	var/high_impact_ruleset_executed = NONE
```

Comment:
You need to change the documentation for this to include that it is a bitfield, and then link to the flags it can use.